net/ipv4/tcp_output.c

```diff
@@ -2723,18 +2720,18 @@ static bool tcp_write_xmit(struct sock *sk, unsigned int mss_now, int nonagle,
 				limit = TLS_MAX_PAYLOAD_SIZE;
 			else
 				limit -= TLS_MAX_OVERHEAD;
+			result = sk->sk_prepare_xmit(sk, skb, mss_now, &limit,
+						     &nskbs);
```

Comment:
fixed, but i don't think that tempesta_skb_get_cb_val is good. Since we add our own struct to unioun with skb dev, we can implement and use skb_tfw_flags